treeherder/etl/bugzilla.py

```diff
@@ -196,9 +196,11 @@ def run(self):
 
             if bug_list:
                 if duplicate_chain_length == 0:
-                    Bugscache.objects.filter(
-                        modified__lt=year_ago, bugzilla_id__isnull=False
-                    ).delete()
+                    Bugscache.objects.exclude(summary="(no bug data fetched)").exclude(
+                        bugzilla_id__in=BugJobMap.objects.distinct("bug__bugzilla_id").values_list(
+                            "bug__bugzilla_id", flat=True
+                        )
```

Comment:
why are you excluding bugs in `bug__bugzilla_id`

Reply:
This excludes all bugs still used for classification in `BugJobMap`. `BugJobMap` instances shall be removed when the associated `Job` gets removed by data cycling.